misc/cfg/gfx_gl_faithful.cfg

```diff
@@ -156,10 +156,12 @@ gl_colorlights               "0"
 
 //Screen & Powerup Blends
 cl_bonusflash                "0"
+cl_demoplay_flash            "1"
```

Comment:
0.33 ?

Reply:
Ah yeah I guess I thought that faithful would have that sent to one - but actually this PR shouldn't do that so I'll change to default.

Reply:
0.33 is default and same in all so removed.